packages/schematics/package_update/utility/npm.ts

```diff
@@ -93,7 +93,7 @@ function _getNpmPackageJson(
       response.on('data', chunk => data += chunk);
       response.on('end', () => {
         try {
-          const json = JSON.parse(data);
+          const json = parseJson(data, JsonParseMode.Loose);
```

Comment:
I don't see why it should be parsed with `Loose` mode. NPM registry is guaranteed to be pure JSON 1.0.